src/index.ts

```diff
@@ -1,15 +1,26 @@
 import * as DCL from 'decentraland-ecs'
-const typesMap = require('decentraland-ecs/types/dcl/decentraland-ecs.api')
 
 type Component = {
   data?: DCL.ObservableComponent
   entityName?: string
 }
 
+interface KeyValuePair<K, T> {
+  key: K
+  value: T
+}
+
 export default class SceneWriter {
+  private DCL
```

Comment:
types?

Reply:
I will put `any` on both of them since one is a `json` file an the other one are all the exports of decentraland.ecs (It doesn't work using `DCL` as type after importing is as wildcard)

Reply:
does `typeof DCL` work?

Reply:
`typeof DCL` -> `object`